### ISSUE TITLE
'bat cache' still takes precedence over existing files.

### ISSUE THREAD

**Issue by eth-p** [sharkdp/bat#666]
Example:

```shell
touch cache
touch test
bat cache test
```

Expected: Both files would be printed.
Actual:

```
error: Found argument 'test' which wasn't expected, or isn't valid in this context

USAGE:
    bat cache [OPTIONS]

For more information try --help
```

This still occurs in bat 0.12.1.

**Comment by sharkdp** [comment]
I was about to write: *"this is the last nail in the coffin for the `bat cache` subcommand"*.
But there is a simple fix for this, see #669.

At least the corner cases are getting more involved... but seriously, if someone has a good solution (see discussion in #584), let's discuss alternative clean solutions again. Otherwise, this is probably going to haunt us forever (the "666" issue number seems fitting, I guess :smile:).

**Comment by sharkdp** [comment]
This has been fixed in [bat 0.13](https://github.com/sharkdp/bat/releases/tag/v0.13.0).
